### PR TITLE
[#481/#482] Error snackbar in Create User page and remove View Users button in homepage

### LINKED ISSUE #481: Use error snackbar in Create User page instead of dialog

### PR DESCRIPTION
**Pull request by sharon-ho**
# General info

**Issue number**: closes #481 and #482

**Task description**: 
- Replace error dialog with snackbar in create user page
- Remove breadcrumb to Admin page that doesn't exist
- Remove View Users button in homepage

# Testing

**Test coverage**: 100%

# Additional notes

**Note to reviewers**:
N/A

**To do before merging**:
N/A

### CODE REVIEW

File: frontend/src/admin/createUser/CreateUserPageContents.jsx [hunk 1]
```diff
@@ -1,20 +1,12 @@
...
-    this.state = {
-      dialogMessage: ""
-    };
-    this.dialogElement = React.createRef();
     this.handleSubmit = this.handleSubmit.bind(this);
-    this.openDialog = this.openDialog.bind(this);
-    this.setDialogMessage = this.setDialogMessage.bind(this);
   }
 
   handleSubmit(username, password) {
```
Comment (felixlapierre) [inline frontend/src/admin/createUser/CreateUserPageContents.jsx]: Can you get rid of the code smell? Perhaps by moving the function out of the class.

File: frontend/src/admin/createUser/CreateUserPage.jsx [hunk 2]
```diff
@@ -6,7 +6,6 @@ import CreateUserPageContents from "./CreateUserPageContents";
 export default function CreateUserPage() {
   const breadcrumbs = [
     ["Home", "/Home"],
-    ["Admin", "/Admin"],
```
Comment (felixlapierre) [inline frontend/src/admin/createUser/CreateUserPage.jsx:9]: If we can have breadcrumbs without links I would keep this breadcrumb. I feel like it emphasizes that this is an admin action.

File: frontend/src/admin/__tests__/CreateUserPageContents.test.jsx [hunk 3]
```diff
@@ -45,46 +35,33 @@ describe("<CreateUserPageContents/> class component", () => {
 
   describe("render() returns a component that", () => {
     it("contains one <CreateUserFormConsole/> component with handleSubmit passed as prop", () => {
```
Comment (felixlapierre) [inline frontend/src/admin/__tests__/CreateUserPageContents.test.jsx]: Test name no longer accurate (there is no CreateUserFormConsole in here anymore)

File: frontend/src/admin/__tests__/CreateUserPageContents.test.jsx [hunk 4]
```diff
@@ -45,46 +35,33 @@ describe("<CreateUserPageContents/> class component", () => {
...
+
+      const formConsoleProps = form.props();
+
+      expect(formConsoleProps.handleSubmit).toBe(expected.handleSubmit);
+      expect(formConsoleProps.buttonName).toEqual(expected.buttonName);
+      expect(formConsoleProps.isValidate).toBe(expected.isValidate);
+      expect(formConsoleProps.passwordError).toEqual(expected.passwordError);
+      expect(formConsoleProps.passwordInputProps).toEqual(
+        expected.passwordInputProps
       );
```
Comment (felixlapierre) [inline frontend/src/admin/__tests__/CreateUserPageContents.test.jsx]: Shorten this to `expect(formConsoleProps).toMatchObject(expected)`

File: frontend/src/admin/__tests__/CreateUserPageContents.test.jsx (hunk 3 above)
Comment (sharon-ho) [reply frontend/src/admin/__tests__/CreateUserPageContents.test.jsx]: 06019e5

File: frontend/src/admin/__tests__/CreateUserPageContents.test.jsx (hunk 4 above)
Comment (sharon-ho) [reply frontend/src/admin/__tests__/CreateUserPageContents.test.jsx]: 8659c3c

File: frontend/src/admin/createUser/CreateUserPageContents.jsx (hunk 1 above)
Comment (sharon-ho) [reply frontend/src/admin/createUser/CreateUserPageContents.jsx]: 8659c3c

File: frontend/src/admin/createUser/CreateUserPage.jsx (hunk 2 above)
Comment (sharon-ho) [reply frontend/src/admin/createUser/CreateUserPage.jsx:9]: 4ec1a5b

Comment (sharon-ho) [reply frontend/src/admin/createUser/CreateUserPage.jsx:9]: Breadcrumb will still be clickable but it won't redirect to anything.

**Review by gdzooks — APPROVED** [review]

**Review by jun-loke — APPROVED** [review]
